src/org/labkey/targetedms/query/ReplicateManager.java

```diff
@@ -52,6 +53,21 @@ public static SampleFile getSampleFile(long sampleFileId)
             .getObject(sampleFileId, SampleFile.class);
     }
 
+    public static SampleFile getNewestSampleFile(Container container)
```

Comment:
where is this called? I am not seeing it used in this PR.

Reply:
I removed the caller when I realized that the client already had the first/last dates available from an existing API call. I deleted this method.